FunctionalTableData/CollectionView/FunctionalCollectionData.swift

```diff
@@ -36,9 +36,9 @@ public class FunctionalCollectionData: NSObject {
 		}
 	}
 	
-	private func dumpDebugInfoForChanges(_ changes: TableSectionChangeSet, previousSections: [TableSection], visibleIndexPaths: [IndexPath], exceptionReason: String?) {
+	private func dumpDebugInfoForChanges(_ changes: TableSectionChangeSet, previousSections: [TableSection], visibleIndexPaths: [IndexPath], exceptionReason: String?, exceptionUserInfo: [AnyHashable: Any]?) {
 		guard let exceptionHandler = FunctionalTableData.exceptionHandler else { return }
-		let exception = FunctionalTableData.Exception(name: name, newSections: sections, oldSections: previousSections, changes: changes, visible: visibleIndexPaths, viewFrame: collectionView?.frame ?? .zero, reason: exceptionReason)
```

Comment:
Why AnyHashale?

Reply:
it's the default `userInfo` type I believe that you get from exceptions

Reply:
True, but do we need it tho? You are only creating dictionaries with keys as string `String:Any`

Reply:
Sure - was just being general, I can change that.

Reply:
Actually, since the way we're passing info back and forth in FTD and FCD is via a native exception, which passes its own `userInfo`, I think it's better that we just adhere to its type without having to convert back and forth.